views/snippets/form_error_multiple_show_errors.html

```diff
@@ -0,0 +1,59 @@
+<div class="error-summary" role="group" aria-labelledby="error-summary-heading-example-2" tabindex="-1">
+
+  <h1 class="heading-medium error-summary-heading" id="error-summary-heading-example-2">
+    Message to alert the user to a problem goes here
+  </h1>
+
+  <p>
+    Optional description of the error(s) and how to correct them
+  </p>
+
+  <ul class="error-summary-list">
+    {{^fullName}}
+    <li><a href="#error-full-name">Descriptive link to the question with an error</a></li>
+    {{/fullName}}
+    {{^niNo}}
+    <li><a href="#error-ni-number">Descriptive link to the question with an error</a></li>
+    {{/niNo}}
+  </ul>
+
+</div>
+
+<h1 class="heading-large form-title">
+  Your details
+</h1>
+
+<div class="form-group error">
+
+  <label class="form-label-bold" for="error-example-full-name">
+
+    Full name
+
+    <span class="form-hint">As shown on your birth certificate or passport</span>
+    <span class="error-message" id="error-full-name">Error message about full name goes here</span>
```

Comment:
Hi @gavboulton,

No, that was a mistake.

I've added the missing aria-describedby attributes and the link is now (and should be) to the label, in the same way as the radio button example links to the legend.